Cooper/Controllers/GameReviewController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Cooper.Models;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace Cooper.Controllers
+{
+    [Route("api/[controller]")]
```

Comment:
Different format of Controller's route. Games have "api/games", here you have "api/[controller]" choose a single way of providing a route